styleguide/_static/issue-templates/feature.md

```diff
@@ -0,0 +1,47 @@
+---
+name: ✨ Feature
```

Comment:
Don't put emojis into anything you might want to group or search later.

Reply:
good point, didn't have that on my radar before

Reply:
@redcatbear  could you elaborate a bit more on this? I don't see a big issue here `name` will be changed by the issue creator to the desired issue name. If someone want's to filter `features` (group by feature) they should filter based on `labels`, issue naming is not guaranteed to be a very stable or solid when it comes to categorization from my point of view.

Reply:
True, labels are for grouping. But most of the time I search for something in a ticket tracker, I search free-form text, since I want to find similar stuff. My observation is that using search fields and APIs gets harder once extended UTF characters (like emojis) come into play. That's why I can accept them for decoration, but I would be careful not to put them into issue titles for example.